http.go

```diff
@@ -106,6 +117,45 @@ func InstrumentHandlerFunc(label string, f func(http.ResponseWriter, *http.Reque
 	return InstrumentHandler(label, http.HandlerFunc(f))
 }
 
+// traceSampler is a time-based request sampler for tracing.
+//
+// It allows at most one request to be traced during each time interval.
+type traceSampler struct {
```

Comment:
Perhaps not in this change, but we should add a test for traceSampler.

Reply:
Ack.